src/main/java/org/jabref/gui/LibraryTab.java

```diff
@@ -90,7 +92,7 @@ public class LibraryTab extends Tab {
     private final BooleanProperty changedProperty = new SimpleBooleanProperty(false);
     private final BooleanProperty nonUndoableChangeProperty = new SimpleBooleanProperty(false);
 
-    private BibDatabaseContext bibDatabaseContext;
+    private ObjectProperty<BibDatabaseContext> bibDatabaseContext;
```

Comment:
Am i overlooking something? Why does this have to be a Property? is there any time the bibDatabase represented by the LibraryTab needs to be swapped?